server/segment_rpc.go

```diff
@@ -210,6 +210,24 @@ func getPaymentSender(payment net.Payment) ethcommon.Address {
 	return ethcommon.BytesToAddress(payment.Sender)
 }
 
+func makeFfmpegVideoProfiles(protoProfiles []*net.VideoProfile) []ffmpeg.VideoProfile {
+	profiles := make([]ffmpeg.VideoProfile, 0, len(protoProfiles))
+	for _, profile := range protoProfiles {
+		name := profile.Name
+		if name == "" {
+			name = "net_" + common.DefaultProfileName(int(profile.Width), int(profile.Height), int(profile.Bitrate))
+		}
+		prof := ffmpeg.VideoProfile{
+			Name:       name,
+			Bitrate:    fmt.Sprintf("%dk", profile.Bitrate),
+			Framerate:  uint(profile.Fps),
```

Comment:
I think we should change `profile.Fps` to `uint32` (right now passing -1 in `profile.Fps` will lead `Framerate` to be equal 4294967295).

Reply:
Good catch! That's something that would need to also be changed in LPMS. Do you mind if we make a separate ticket for this and tackle it in a separate PR?

Reply:
https://github.com/livepeer/lpms/issues/162